src/Emanote/Model/Stork/Index.hs

```diff
@@ -91,18 +102,57 @@ data File = File
   }
   deriving stock (Eq, Show)
 
-fileCodec :: TomlCodec File
-fileCodec =
-  File
-    <$> Toml.string "path"
-      .= filePath
-    <*> Toml.text "url"
-      .= fileUrl
-    <*> Toml.text "title"
-      .= fileTitle
+data Handling
+  = Handling_Ignore
+  | Handling_Omit
+  | Handling_Parse
+  deriving stock (Eq, Show, Generic)
 
-inputCodec :: TomlCodec Input
-inputCodec =
-  Input
-    <$> Toml.list fileCodec "input.files"
-      .= inputFiles
+instance Default Handling where
+  def = Handling_Omit
+
+instance FromJSON Handling where
+  parseJSON = genericParseJSON handlingJSONOptions
+    where
+      handlingJSONOptions :: Aeson.Options
+      handlingJSONOptions =
+        Aeson.defaultOptions
+          { Aeson.constructorTagModifier = toString . T.toLower . T.replace "Handling_" "" . toText
+          }
```

Comment:
This instance can be simplified using 

https://hackage.haskell.org/package/aeson-casing-0.2.0.0/docs/Data-Aeson-Casing.html

Reply:
@jfpedroza Wanna take a stab at this? 

Otherwise, I'll do the rest of the changes (including this one) tomorrow and merge.

Reply:
Yes, I'll give it a try.

Reply:
The package works on `fieldLabelModifier`, not `constructorTagModifier` so I can't use the public functions. And even with the internal functions, they don't work on the form `Handling_Ignore`, they expect camel case.

The only thing useful is `applyFirst` that allows you to do

```haskell
Aeson.constructorTagModifier = applyFirst toLower . drop 9
```

It's not worth to add the package for that function. I added it and changed `replace` with `drop` since it avoids the converting to text and back dance.

Reply:
Right. 

I actually should have suggested https://hackage.haskell.org/package/deriving-aeson because that's exactly what we need here.

cf. https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/deriving_via.html